README.template.md

```diff
@@ -51,9 +51,53 @@ Here is an example to get you started:
 {{ usageExampleCode }}
 ```
 
+In addition to the `SimpleCacheClient` client library shown in the example above, a PHP PSR-16 "SimpleCache"
+implementation is also included in the SDK. See the [PSR-16 client README](README-PSR16.md) for more details.
```

Comment:
Do you think it might be worth pushing this up into the `Examples` section to make it more visible?

Reply:
Good idea, I'm moving it up.